content/docs/reference/glossary.md

```diff
@@ -273,7 +273,7 @@ Older projects may have a `web-access` system role, used by the [SQL Editor](#sq
 
 Each Neon project is created with a primary branch called `main`, but you can designate any branch as your project's primary branch. The advantage of the primary branch is that its compute endpoint remains accessible if you exceed your project's limits, ensuring uninterrupted access to data that resides on the primary branch. For more information, see [Primary branch](/docs/manage/branches#primary-branch).
 
-### Pro plan
+### Pro Plan
```

Comment:
Should this be "Neon Pro Plan" like elsewhere?

Reply:
Fixed. Thank you